service/src/integrationTest/resources/fixtures/C21CaseData.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Can we not re-use callback-request.json?

Reply:
Oh... this didn't exist when I was working on my branch. I used the C6CaseData for reference (which seems to have gone now). I'll change it to use callback-request, thanks